udata_hydra/cli.py

```diff
@@ -137,7 +137,7 @@ async def crawl_url(url: str, method: str = "get"):
 
 
 @cli
-async def check_resource(resource_id: str, method: str = "get"):
+async def check_resource(resource_id: str, method: str = "get", force_analysis: bool = False):
```

Comment:
I would even set `force_analysis` to True by default? I think if we're using a CLI to launch a check, I would say we probably want the full run